### PR TITLE
Implement evolution chain

### PR DESCRIPTION
**Pull request by piotr-wawro**

### CODE REVIEW

File: src/screens/details/Evolution.js [hunk 1]
```diff
@@ -1,17 +1,69 @@
...
 
+  var tier1 = chain;
+  var tier2 = chain.evolvesTo;
+  var tier3;
+
+  if (tier2.length == 1) {
+    tier3 = tier2[0].evolvesTo;
+  } else {
+    tier3 = [];
+  }
```
Comment (kamil-matula) [inline src/screens/details/Evolution.js]: You can write it in one line: `var tier3 = tier2.length == 1 ? tier2[0].evolvesTo : [];`

**Review by kamil-matula — APPROVED** [review]
It looks really pretty and works perfectly, so good job!

File: src/screens/details/Evolution.js [hunk 2]
```diff
@@ -1,17 +1,63 @@
 import React from 'react';
-import { SafeAreaView, StyleSheet, Text } from 'react-native';
+import { SafeAreaView, StyleSheet, Dimensions, View, ScrollView } from 'react-native';
 import { useTheme } from 'react-native-paper';
 
 import { DetailsAppBar } from '../../components';
+import EvolutionArrow from '../../components/EvolutionArrow';
+import EvolutionCard from '../../components/EvolutionCard';
```
Comment (Blazevarjo) [inline src/screens/details/Evolution.js:7]: ``
import { DetailsAppBar, EvolutionArrow, EvolutionCard  } from '../../components';
``

File: src/screens/details/Evolution.js [hunk 3]
```diff
@@ -1,17 +1,63 @@
...
 
 export default function Evolution({ color, sprite, chain }) {
   const colors = useTheme().colors;
 
+  var tier1 = chain;
+  var tier2 = chain.evolvesTo;
+  var tier3 = tier2.length == 1 ? tier2[0].evolvesTo : [];
```
Comment (Blazevarjo) [inline src/screens/details/Evolution.js:14]: var => const

**Review by Blazevarjo — COMMENTED** [review]
Everything seems fine, only slight refactor. gj

**Pull request merged**